vspace/src/main/webapp/resources/extra/Home.css

```diff
@@ -1,768 +1,769 @@
+    position: absolute;
+    overflow: visible;
+    width: 21px;
+    height: 21px;
+    left: 6px;
+    padding-top: 4px;
+}
+
+.Icon_awesome_info_module_m {
+    overflow: visible;
+    position: absolute;
+    font-size: 20px;
+    top: 4px;
+    left: 4px;
+    color: rgba(150, 45, 62, 1);
+}
+
+.imageStyle {
+    min-height: 500px;
+    min-width: 800px;
+    border-radius: 13px;
+}
+
+.spaceName {
+    font-size: 25px;
+}
+
+.Icon_awesome_info_staff_c {
+    overflow: visible;
+    position: absolute;
+    font-size: 20px;
+    top: 4px;
+    left: 4px;
+}
+
+.Icon_awesome_info_staff_m {
+    overflow: visible;
+    position: absolute;
+    font-size: 20px;
+    top: 4px;
+    left: 4px;
+    color: rgba(150, 45, 62, 1);
+}
+
+.Icon_awesome_info_staff_e {
+    overflow: visible;
+    position: absolute;
+    font-size: 20px;
+    top: 4px;
+    left: 4px;
+    color: rgba(150, 45, 62, 1);
+}
+
+.form_input {
+	position: relative;
+	z-index: 3;
+}
+
+.searchBar {
+    position: absolute;
+    pointer-events: auto;
+    cursor: pointer;
+    z-index: 5;
+    right: 0;
+    background: none;
+}
+
+.form_input:placeholder-shown {
+	z-index: 1;
+}
+
+.link-modal {
+	display: none;
+	position: fixed;
+	top: 2.5%;
+	right: 2.5%;
+	left: 2.5%;
+	bottom: 0;
+	z-index: 9999;
+	overflow-y: scroll;
+	background: white;
+	padding: 25px;
+	border-radius: 10px;
+	box-shadow: 0 0 300px rgba(0,0,0,0.9);
+}
+
+.divider {
+    height: 100%;
+    width: 1px;
+    background: var(--med-grey);
+    margin: 0px 20px;
+}
+
+.Icon_awesome_info_module_m_sidenav {
+    overflow: visible;
+    position: absolute;
+    font-size: 20px;
+    left: 6px;
+    color: rgba(150, 45, 62, 1);
+}
+
+.Ellipse_9_module_c_sidenav {
+    filter: drop-shadow(3px 3px 6px rgba(0, 0, 0, 0.161));
+    position: absolute;
+    overflow: visible;
+    width: 21px;
+    height: 21px;
+    left: 6px;
+    padding-top: 0px;
+}
+
+.Ellipse_10_module_c_sidenav {
+    position: absolute;
+    overflow: visible;
+    width: 25px;
+    height: 25px;
+    left: 4px;
+    padding-top: 0px;
+    margin-top: -2px;
+}
+
+.Ellipse_8_module_c_sidenav {
+    position: absolute;
+    overflow: visible;
+    width: 29px;
+    height: 29px;
+    left: 2px;
+    margin-top: -3px;
+}
+
+.textDivStaff {
+    position: relative;
+    top: 0px;
+    overflow: visible;
+    text-align: justify;
+}
+
+.choiceDiv {
+    position: relative;
+    top: 0px;
+    overflow: visible;
+    text-align: justify;
+}
+.custom_form_style {
+    display: none;
+    width: 25%;
+}
+
+.circle-btn {
+	background: white;
+	width: 48px;
+	height: 48px;
+	border-radius: 50%;
+	box-shadow: 0px 5px 10px rgba(0,0,0,0.2);
+}
+
+.slide-nav {
+    left: 10%;
+    position: absolute;
+    top: 10%;
+}
+
+.circle-btn-slide-nav {
+    background: white;
+    width: 33px;
+    height: 33px;
+    left:19%;
+    border-radius: 50%;
+    box-shadow: 0px 5px 10px rgba(0,0,0,0.2);
+}
+
+.circle-btn:hover {
+	background: #f2f2f2;
```

Comment:
next time, make sure to note whenever you change the line ending, so it's clear why the whole file shows up as changed.